src/containers/SignUp/style.scss

```
@@ -0,0 +1,41 @@
+@import url('https://fonts.cdnfonts.com/css/vantage-brk');
+
+.signup {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.signup-box{
+  width: 500px;
+  background: #f9d859;
+  padding: 15px;
+  border: 3px solid rgb(150, 150, 150);
+  box-shadow: 0px 0px 0px 1px rgb(90, 90, 90),
+    inset 0px 0px 0px 1px rgb(90, 90, 90),
+    1px 2px 0px 2px rgb(100, 100, 100);
+  border-radius: 10px;
+  align-items: center;
+  position: absolute;
+  top: 40%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  .box-header {
```

```suggestion
  
  .box-header {
```
-- Also, you are inside the other class. You should follow a standard to use BEM and be able to use &__